### PR TITLE
Add example of using AVA

### PR DESCRIPTION
**Pull request by ragingwind**
Related to #10

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -111,6 +111,69 @@ describe('application launch', function () {
...
+
+```js
+'use strict';
+
+import test from 'ava';
+import {Application} from 'spectron';
+import path from 'path';
+
+test.beforeEach(t => {
+	t.context.app = new Application({
```
Comment (kevinsawicki) [inline README.md]: Looks like this is using hard tabs, mind using a two space indent to match the other code blocks in this file?

Comment (ragingwind) [inline README.md]: Oh I see

**Comment by ragingwind** [comment]
done.

File: README.md [hunk 2]
```diff
@@ -111,6 +111,69 @@ describe('application launch', function () {
...
 ```
 
+### With AVA
+
+Spectron also works with [AVA](https://github.com/sindresorhus/ava) that allows you to write your test code in ES2015 without extra support.
+
+```js
+'use strict';
+
+import test from 'ava';
```
Comment (kevinsawicki) [inline README.md]: Is babel required to use this syntax or is there a version of node that supports the `import` syntax?

Comment (kevinsawicki) [inline README.md]: Or is this syntax support built-in automatically to AVA?

Comment (kevinsawicki) [inline README.md]: Nevermind, I found https://github.com/sindresorhus/ava#es2015-support which explains it.

**Comment by kevinsawicki** [comment]
Thanks for this :+1:

**Pull request merged**